### PR TITLE
fix: added color theme data for the app

### PR DESCRIPTION
**Pull request by DesmondTambe**
## Description

- Updates to color themeData

## Type of Change

- [x] ✨ New feature (non-breaking change which adds functionality)
- [x] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [x] 🗑️ Chore

## Pre-launch Checklist

- [ ] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ ] I read and ran all relevant commands as specififed in the Running Tests section of the [Contributor Guide].
- [ ] The title of the PR follows the [Conventional Commits] guideline
- [ ] My local branch follows the naming standards in the [Deepsource Branch Naming Convention] or [Biodiversity Branch Naming Convention]
- [ ] I listed at least one issue that this PR fixes in the description above.
- [ ] I updated `pubspec.yaml` with an appropriate new version according to the [pub versioning philosophy],
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] I added new tests to check the change I am making, or this PR is [test-exempt].
- [ ] All existing and new tests are passing.

[Contributor Guide]: https://github.com/FlutterPlaza/.github/blob/main/CONTRIBUTING.md
[Conventional Commits]: https://www.conventionalcommits.org/en/v1.0.0-beta.4/
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[Biodiversity Branch Naming Convention]: https://bit.ly/3DyYSwM
[Deepsource Branch Naming Convention]: https://bit.ly/3Y08Gs4

### CODE REVIEW

File: lib/home/view/widgets/navbar_header.dart [hunk 1]
```diff
@@ -42,7 +44,11 @@ class NavHeader extends StatelessWidget {
               children: [
                 showSearchIcon
                     ? InkWell(
-                        onTap: () => print('Navigate to search scren'),
+                        onTap: () {
+                          context.pushRoute(
```
Comment (lkmandy) [inline lib/home/view/widgets/navbar_header.dart:48]: Why does the search icon lead to the LatesActivities screen instead of the Search screen?

Comment (DesmondTambe) [inline lib/home/view/widgets/navbar_header.dart:48]: I was just testing out the Nested navigation

**Comment by lkmandy** [comment]
@DesmondTambe I just tested it out locally and it looks good.

**Review by lkmandy — APPROVED** [review]

**Comment by lkmandy** [comment]
LGTM

**Comment by DesmondTambe** [comment]
> LGTM

@lkmandy Thanks Amanda... There will be subsequent update for other widgets

**Comment by lkmandy** [comment]
Right,
@TikuGaelle needs this merged in order to complete her work, so we'll prioritize that.

File: lib/home/view/widgets/bottom_nav_bar.dart [hunk 2]
```diff
@@ -49,11 +51,12 @@ class _BottomNavBarState extends State<BottomNavBar> {
                     )
                   : Icon(
                       FpbIcons.home,
-                      color: colors.onSurface,
+                      color: Colors.grey[500],
```
Comment (fonkamloic) [inline lib/home/view/widgets/bottom_nav_bar.dart:54]: This color should be dynamic from the themedata

File: lib/home/view/widgets/bottom_nav_bar.dart [hunk 3]
```diff
@@ -79,11 +82,12 @@ class _BottomNavBarState extends State<BottomNavBar> {
                     )
                   : Icon(
                       FpbIcons.wallet,
-                      color: colors.onSurface,
+                      color: Colors.grey[500],
```
Comment (fonkamloic) [inline lib/home/view/widgets/bottom_nav_bar.dart:85]: Same here

File: lib/home/view/widgets/bottom_nav_bar.dart (hunk 2 above)
Comment (DesmondTambe) [reply lib/home/view/widgets/bottom_nav_bar.dart:54]: Alright, great notice. thanks 

Fixed!

**Pull request merged**